### PR TITLE
Improvement for detecting Yiiframework and Added PHP Debug Bar

### PR DESCRIPTION
**Pull request by professor93**

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -1923,6 +1923,20 @@
 			"icon": "Debian.png",
 			"website": "http://debian.org"
 		},
+    "PHPDebugBar": {
+			"cats": [
+				"47"
+			],
+			"env": [
+				"^phpdebugbar",
```
Comment (jvoisin) [inline src/apps.json]: Regexp aren't case-sensitive.

File: src/apps.json [hunk 2]
```diff
@@ -1923,6 +1923,20 @@
...
+				"^PhpDebugBar"
+			],
+			"script": [
+				"debugbar.*\\.js"
```
Comment (jvoisin) [inline src/apps.json]: This would match stuff like `https://mywebsite.com/my-article-about-debugbar/script/bootstrap.js`.

File: src/apps.json [hunk 3]
```diff
@@ -9366,11 +9380,13 @@
 			"cats": [
 				"18"
 			],
+			"env": "^yii$",
```
Comment (jvoisin) [inline src/apps.json]: This will generator false-positives with javacript compressors.

File: src/apps.json [hunk 4]
```diff
@@ -9366,11 +9380,13 @@
...
 			"html": [
 				"Powered by <a href=\"http://www.yiiframework.com/\" rel=\"external\">Yii Framework</a>",
 				"<input type=\"hidden\" value=\"[a-zA-Z0-9]{40}\" name=\"YII_CSRF_TOKEN\" \\/>",
 				"<!\\[CDATA\\[YII-BLOCK-(?:HEAD|BODY-BEGIN|BODY-END)\\]"
 			],
+			"script": "yii.*\\.js",
```
Comment (jvoisin) [inline src/apps.json]: This will generator a lot of false positives.

**Comment by AlexByte** [comment]
Change of different technologies should be placed in separate pull requests.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**